
overhead_grasping/grasp_time : 2.0
overhead_grasping/setup_velocity : 0.8 #0.65 #0.40 # 0.40 #0.4 # 0.15

overhead_grasping/static_deriv_thresh : { "accelerometer" : [0.6, 0.6, 0.6],
                    "joint_velocities" : [0.45, 0.45, 0.45, 0.45, 0.45, 0.45, 0.45],
                    "joint_angles" : [0.04, 0.04, 0.04, 0.04, 0.04, 0.03, 0.04],
                    "joint_efforts" : [2.0, 1.0, 0.5, 0.5, 0.5, 1.0, 1.0],
                    "r_finger_periph_pressure" : [100.0], 
                    "r_finger_pad_pressure" : [200.0],
                    "l_finger_periph_pressure" : [100.0],
                    "l_finger_pad_pressure" : [200.0],
                    "gripper_pose" : [0.01, 0.01, 0.01, 100.1, 100.1, 100.1, 100.1]}
